### PR TITLE
Enable GitHub Actions

### PR DESCRIPTION
**Pull request by sirodoht**
This PR adds configuration for GitHub Actions CI setting up the Django app + building vite.js + running pytest.

### CODE REVIEW

**Comment by nicksellen** [comment]
This would be useful to have, as currently we're running blind without the tests running in ci...

**Comment by sirodoht** [comment]
> This would be useful to have, as currently we're running blind without the tests running in ci...

Done! Only took 3 weeks 😅

File: .github/workflows/.build.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        pip install -r requirements.txt
+        pip install -r requirements_dev.txt
+
+    - name: Run Tests
+      run: |
+        export POSTGRES_HOST=localhost
+        export POSTGRES_DB=postgres
+        export POSTGRES_USER=postgres
+        export POSTGRES_PASSWORD=postgres
+        export VITE_DEV_MODE=0
```
Comment (nicksellen) [inline .github/workflows/.build.yml]: Just a nitpick, using `env:` here would seem slightly nicer.

File: .github/workflows/.build.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+name: Build
```
Comment (nicksellen) [inline .github/workflows/.build.yml]: It's more like `Test` than `Build`

**Review by nicksellen — APPROVED** [review]
Great to have! 2 nitpicks for you 😆

File: .github/workflows/.build.yml (hunk 1 above)
Comment (sirodoht) [reply .github/workflows/.build.yml]: nice, changing!

**Pull request merged**